src/index.js

```diff
@@ -15,6 +15,8 @@ const graphql = (document, config = {}) => {
     throw new Error(`graphql(react-apollo-helpers): Expected 1 graphQL operation. ${document.definitions.length} operations detected.`);
   }
 
+  const { name } = config;
```

Comment:
This is a bug fix. It probably belongs in a separate pr, but since it's needed here and is a one-liner, let's maybe not worry about it.